### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by nwithan8**
# Description

- Move Security.cs file to proper directory
- Make sure all models consistently extend base EasyPostObject class
- `myBatch.GenerateLabel()` no longer has a default file format value

# Testing

- No new unit tests required
- All unit tests pass

# Pull Request Type

Please select the option(s) that are relevant to this PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement (fixing a typo, updating readme, renaming a variable name, etc)

### CODE REVIEW

File: EasyPost/Models/API/Shipment.cs [hunk 1]
```diff
@@ -154,7 +154,7 @@ public async Task Buy(Rate rate, string? insuranceValue = null, bool withCarbonO
         /// </summary>
         /// <param name="fileFormat">Format to generate the label in. Valid formats: "pdf", "zpl" and "epl2".</param>
         [CrudOperations.Update]
-        public async Task<Shipment> GenerateLabel(string fileFormat)
+        public async Task<Shipment> GenerateLabel(string fileFormat = "pdf")
```
Comment (Justintime50) [inline EasyPost/Models/API/Shipment.cs:157]: Question: Why have a default instead of requiring users to specify?

**Review by Justintime50 — COMMENTED** [review]
Do we need to re-record these cassettes? No data changed, the build should have passed without re-recording them.

**Comment by nwithan8** [comment]
> Do we need to re-record these cassettes? No data changed, the build should have passed without re-recording them.

Collateral damage as I was experimenting with changing the HTTP method.

File: EasyPost/Models/API/Shipment.cs (hunk 1 above)
Comment (nwithan8) [reply EasyPost/Models/API/Shipment.cs:157]: We can do that too. We have a default on batch generate label, so just want to remain consistent either way.

Comment (Justintime50) [reply EasyPost/Models/API/Shipment.cs:157]: I don't love that we have one on batch generate label... I'd prefer we didn't include them but we can't just rip out the batch one.

Do we do this in any of our other libs?

**Comment by Justintime50** [comment]
Let's reset the cassettes so they aren't tracked in this PR if possible.

File: EasyPost/Models/API/Shipment.cs (hunk 1 above)
Comment (nwithan8) [reply EasyPost/Models/API/Shipment.cs:157]: Java and Python have no default, so we can go with that.

File: EasyPost/Models/API/Batch.cs [hunk 2]
```diff
@@ -89,7 +89,7 @@ public async Task<Batch> Buy()
         /// </summary>
         /// <param name="fileFormat">Format to generate the label in. Valid formats: "pdf", "zpl" and "epl2".</param>
         [CrudOperations.Update]
-        public async Task<Batch> GenerateLabel(string fileFormat = "pdf")
```
Comment (Justintime50) [inline EasyPost/Models/API/Batch.cs:92]: Issue: This is a breaking change, we can't rip out a default here. 

What I meant was although sad and not consistent, let's not add it to the Shipment function. We could note this in code with a TODO to remove the default in an upcoming major release instead?

**Review by Justintime50 — CHANGES_REQUESTED** [review]

File: EasyPost/Models/API/Batch.cs (hunk 2 above)
Comment (nwithan8) [reply EasyPost/Models/API/Batch.cs:92]: Sounds good

**Review by Justintime50 — APPROVED** [review]

**Pull request merged**